dynawo/sources/Models/Modelica/Dynawo/Electrical/Machines/SignalN/GeneratorPVRemote.mo

```diff
@@ -0,0 +1,77 @@
+within Dynawo.Electrical.Machines.SignalN;
+
+/*
+* Copyright (c) 2015-2020, RTE (http://www.rte-france.com)
+* See AUTHORS.txt
+* All rights reserved.
+* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, you can obtain one at http://mozilla.org/MPL/2.0/.
+* SPDX-License-Identifier: MPL-2.0
+*
+* This file is part of Dynawo, an hybrid C++/Modelica open source time domain simulation tool for power systems.
+*/
+
+model GeneratorPVRemote "Model for generator PV based on SignalN for the frequency handling and a remote voltage regulation"
+
+  import Dynawo.Electrical.Machines;
+  extends Machines.BaseClasses.BaseGeneratorSimplified;
+  extends AdditionalIcons.Machine;
+
+  type QStatus = enumeration (Standard "Reactive power is fixed to its initial value",
+                              AbsorptionMax "Reactive power is fixed to its absorption limit",
+                              GenerationMax "Reactive power is fixed to its generation limit");
+
+public
+
+  parameter Types.VoltageModulePu URef0Pu "Start value of the voltage regulation set point in p.u (base UNom)";
```

Comment:
protected

Reply:
Same as before, as I give this parameter in the .par, can it be protected ?